rln/src/circuit.rs

```diff
@@ -33,8 +33,8 @@ const WASM_FILENAME: &str = "rln.wasm";
 // Note that the circuit and keys in TEST_RESOURCES_FOLDER are compiled for Merkle trees of height 20 & 32
 // Changing these parameters to other values than these defaults will cause zkSNARK proof verification to fail
 pub const TEST_PARAMETERS_INDEX: usize = 0;
-pub const TEST_TREE_HEIGHT: usize = [20, 32][TEST_PARAMETERS_INDEX];
-pub const TEST_RESOURCES_FOLDER: &str = ["tree_height_20", "tree_height_32"][TEST_PARAMETERS_INDEX];
+pub const TEST_TREE_HEIGHT: usize = [20][TEST_PARAMETERS_INDEX];
```

Comment:
you should also remove the tree_height_32 resources from the resources sub-directory.
please fix the comments which reference the tree height 32 too.
we can remove the TEST_TREE_HEIGHT, TEST_PARAMETERS_INDEX constants since they can have only one value.

Reply:
Removed resources.
Removed `TEST_PARAMETERS_INDEX `, but left `TEST_TREE_HEIGHT`, because it is used all over the place, and removing it would make it less descriptive.